m4/ctng_python.m4

```diff
@@ -63,6 +63,11 @@ AC_DEFUN([CTNG_PYTHON],
   for python in python3.13 python3.12 python3.11 python3.10 python3.9 python3.8 python3.7 dnl
 python3.6 python3.5 python3.4 python3.3 python3.2 python3.1 python3.0 python2.7 dnl
 python2.6 python2.5 python2.4 python2.3 python2.2 python2.1 python; do
+    AS_UNSET(ac_cv_path_PYTHON_BIN)
+    AS_UNSET(PYTHON_BIN)
+    AS_UNSET(ac_cv_prog_PYTHON_BIN_)
+    AS_UNSET(PYTHON_BIN_)
+
```

Comment:
Hmm I'm a bit dubious about this. Seems to invalidate the purpose of having a cache. I think the right thing is probably to do what I hinted at in https://github.com/crosstool-ng/crosstool-ng/commit/40c7e23be89bfb47fbae28e1db46f524bf5b8ed0 and use the upstream AX_PYTHON and come up with a better way of getting the path.